Sources/DashX/Constants.swift

```diff
@@ -5,6 +5,8 @@ struct Constants {
     static let USER_PREFERENCES_KEY_ANONYMOUS_UID = "\(PACKAGE_NAME).anonymous_uid"
     static let USER_PREFERENCES_KEY_DEVICE_TOKEN = "\(PACKAGE_NAME).device_token"
     static let USER_PREFERENCES_KEY_BUILD = "\(PACKAGE_NAME).build"
+    static let USER_PREFERENCES_ACCOUNT_UID = "\(PACKAGE_NAME).account_uid"
```

Comment:
- Follow a consistent naming convention (missing the KEY_ part)
- Shift these to be ordered with the other keys (account_uid, account_anonymous_uid, identity_token, device_token, *then* build)